flow/project.py

```diff
@@ -1563,9 +1737,9 @@ def _get_operations_status(self, job, cached_status):
         starting_dict = functools.partial(dict, scheduler_status=JobStatus.unknown)
         status_dict = defaultdict(starting_dict)
         for group in self._groups.values():
-            completed = group.complete(job)
-            eligible = False if completed else group.eligible(job)
-            scheduler_status = cached_status.get(group._generate_id(job),
+            completed = group._complete([job])
+            eligible = False if completed else group._eligible([job])
```

Comment:
This could raise errors if single jobs are not accepted for a group correct? I know that this will not be a problem in this PR, but in the future it could be. I imagine this function is to be removed or unused in the status check PR? If so, than this is fine for now; otherwise, we need to change the function to work with aggregates of jobs.

Reply:
This is not the case because we're going to iterate over `group.aggregate(jobs)` in #336 which will always raise an error if and only if the users have failed to specify the function parameters for the conditions correctly.

Reply:
Okay, as long as this is changed there.